recipes/google-resumable-media/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set version = "0.0.2" %}
+
+package:
+  name: google-resumable-media
+  version: {{ version }}
+
+source:
+  fn: google-resumable-media-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/g/google-resumable-media/google-resumable-media-{{ version }}.tar.gz
+  sha256: c48e88da9f4e876ee3a79c9e872425745ffb05bfc236d1b0d98527fbb6410c82
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+    - six
+
+test:
+  imports:
+    - google
+    - google.resumable_media
+
+about:
+  home: https://github.com/GoogleCloudPlatform/google-resumable-media-python
+  summary: Utilities for Google Media Downloads and Resumable Uploads
+  doc_url: https://googlecloudplatform.github.io/google-resumable-media-python
+  dev_url: https://github.com/GoogleCloudPlatform/google-resumable-media-python
+  license: Apache-2.0
+  license_family: APACHE
+  license_file: {{ RECIPE_DIR }}/LICENSE
```

Comment:
Could you open an issue or (preferably) a PR on the source repo to add the license file to the `MANIFEST.in` and then xref that issue in a comment in the recipe as a reminder to patch it later.